dr-scripts_install.lic

```diff
@@ -0,0 +1,91 @@
+=begin
+
+	dr-scripts install helper
+	Aims to automate running through the steps from https://github.com/rpherbig/dr-scripts/wiki/First-Time-Setup
+	Please see us on Discord if there are any issues, or raise an issue on Github
+
+        author: Mahtra (mahtra@seul.org)
+        game: Dragonrealms
+        version: 1.0
+
+        changelog:
+                1.0 (2022-01-27):
+                        First release
+
+=end
+
+  def wait_for_script_to_complete(name, args = [], flags = {})
+    script_handle = start_script(name, args.map { |arg| arg =~ /\s/ ?  "\"#{arg}\"" : arg }, flags)
+    if script_handle
+      pause 2
+      pause 0.5 while Script.running.include?(script_handle)
+    end
+    script_handle
+  end
+
+  def download_dependency
+    if(File.exist?('./scripts/dependency.lic')) 
+      echo "Depdency exist already, skipping downloads"
+    else
+      echo "Downloading dependency"
+      wait_for_script_to_complete('repository', ['download', 'dependency.lic'])
+      pause 2
+    end
+  end
+  
+  def install_dependency
+    if(File.exist?('./scripts/common.lic')) 
+      echo "Depdency exist already installed, skipping downloads"
+    else
+      echo "Installing dependency, this might take a few minutes"
+      wait_for_script_to_complete('dependency', ['install'])
+    end
+  end
+  
+  def enable_use_lich_fork
+    echo "Switching to DR fork"
+    ExecScript.start('use_lich_fork')
+    pause 2
+  end
+      
+  def run_setupaliases
+    if(File.exist?('./scripts/setupaliases.lic')) 
+       wait_for_script_to_complete('setupaliases')
+       pause 2
+    end
+  end
+    
+  def enable_autostarts
+    echo "Adding textsubs, and roomnumbers to autostarts"
+    ExecScript.start("autostart(['textsubs', 'roomnumbers'])")
```

Comment:
Is this necessary? Personally I don't run with either of these scripts on

Reply:
It's step 9!

Reply:
Actually, these are here because Briar asked for them. The target audience of this script isn't you or me, but brand new installs, and that's the group Briar deals with the most. I'll leave this in. You and I can easily edit this line if we're using the script, where newbies won't be doing that. If it becomes a problem later, we can change it.

Reply:
Makes sense, thanks!